src/test/java/seedu/bookmark/logic/commands/AddCommandTest.java

```diff
@@ -170,19 +170,19 @@ public boolean hasBook(Book book) {
     /**
      * A Model stub that always accept the person being added.
```

Comment:
Thanks for doing the renaming! There's some parts which haven't been renamed such as the Javadoc here, comments in the `equals()` test as well as here: https://github.com/AY2021S1-CS2103T-F13-2/tp/blob/c568a3c5177c68a33c3e6279923fa8f3c0356df0/src/test/java/seedu/bookmark/logic/commands/AddCommandTest.java#L153
Would be great if you could change those too, thanks!

Reply:
Must have missed this part out. Thanks!